animepahe-dl.sh

```diff
@@ -3,10 +3,10 @@
 # Download anime from animepahe using CLI
 #
 #/ Usage:
-#/   ./animepahe-dl.sh [-s <anime_slug>] [-e <episode_num1,num2,num3-num4...>] [-l]
+#/   animepahe-dl.sh [-a <anime>] [-e <episode_num1,num2,num3-num4...>] [-l]
 #/
 #/ Options:
-#/   -s <slug>               Anime slug, can be found in $_ANIME_LIST_FILE
```

Comment:
Please keep `-s <slug>` flag, in case it needs to be used in some case without `fzf` interaction.